MinitweetApi/obj/Debug/net6.0/.NETCoreApp,Version=v6.0.AssemblyAttributes.cs

```diff
@@ -1,4 +1,4 @@
 // <autogenerated />
 using System;
 using System.Reflection;
-[assembly: global::System.Runtime.Versioning.TargetFrameworkAttribute(".NETCoreApp,Version=v6.0", FrameworkDisplayName = "")]
+[assembly: global::System.Runtime.Versioning.TargetFrameworkAttribute(".NETCoreApp,Version=v6.0", FrameworkDisplayName = "")]

```

Comment:
Let us try not to commit these files, add them to gitignore or don't use 'git add -A'. I will approve this, but feel free to update this branch or make a cleanup branch afterwards